packages/core/src/events/index.ts

```diff
@@ -28,7 +28,7 @@ export class EventFactory {
       ...this.baseEvent(),
       event,
       type: 'track',
-      properties,
+      properties: properties ?? {},
```

Comment:
Would this be a breaking change in the eventual `track` playload structure (i.e., after this change track payload will always have a `properties` object regardless )?